src/libs/workspace-utils.ts

```diff
@@ -4,6 +4,12 @@ export const cloudProviderTypes: Record<CloudProviderType, CloudProviderType> =
   AZURE: 'AZURE',
   GCP: 'GCP'
 }
+
+export const cloudProviderLabels: Record<CloudProviderType, string> = {
```

Comment:
Perhaps rebase

Reply:
Yeah, I was just confused because this change was in your last PR.

Reply:
Ah, figured out my confusion. I thought https://github.com/DataBiosphere/terra-ui/pull/3661 had already merged.

Reply:
Ah. I was holding off on #3661 until I checked with product.